src/xgov-dapp/src/shared/csvSigner.ts

```diff
@@ -25,13 +20,13 @@ export async function signCsv(csv: string): Promise<{ signedCsv: SignedCsv[]; pu
   const encoder = new TextEncoder()
   try {
     const signedCsv = await Promise.all(
-      results.data.map((row): Promise<SignedCsv> => {
+      results.data.map((row): Promise<Gate> => {
         return ed
           .signAsync(Uint8Array.from([...algo.decodeAddress(row.address).publicKey, ...encoder.encode(`${row.weight}`)]), privateKey)
```

Comment:
This won't work, it treats it as a utf-8 string rather than a uint64. I have the implementation for this in my PR though